pegasus/sites.v3/code.org/public/learn/index.haml

```diff
@@ -113,6 +115,11 @@ style_min: false
   - tutorial[:string_detail_platforms]              = hoc_s(prefix + "string_detail_platforms")
   - tutorial[:string_detail_programming_languages]  = hoc_s(prefix + "string_detail_programming_languages")
 
+  - if ["mc", "dance"].include? hoc_launch
+    - if tutorial[:short_code] == "mchoc"
```

Comment:
There's also a tutorial in cdo-tutorials that has the short code "aquatic" - what's the difference?

Reply:
We use "mchoc" to go to https://code.org/minecraft and then the tutorials themselves are hidden on https://code.org/learn but have codes such as "aquatic" for internal use.